Userland/Applications/SoundPlayer/PlaybackManager.h

```diff
@@ -12,8 +12,11 @@
 #include <LibAudio/Loader.h>
 #include <LibCore/Timer.h>
 
-#define PLAYBACK_MANAGER_BUFFER_SIZE 48 * KiB
-#define PLAYBACK_MANAGER_RATE 44100
+// Controls the GUI update rate. A smaller value makes the visualizations nicer.
+constexpr u32 playback_manager_update_rate_ms = 50;
+
+// Number of milliseconds of audio data contained in each audio buffer
+constexpr u32 playback_manager_buffer_size_ms = 100;
```

Comment:
Instead of `playback_manager_*`, could we put these inside `PlaybackManager`?
e.g. `PlaybackManager::update_rate_ms`